engine/src/main/java/org/hibernate/validator/constraints/SafeHtml.java

```diff
@@ -81,6 +81,12 @@
 		 * @return list of tag attributes which are whitelisted.
 		 */
 		String[] attributes() default { };
+
+		/**
+		 * @return list of valid protocols.
+		 * @since 5.4.2
```

Comment:
Hum, adding this in a micro is a bit of an issue (micros shouldn't alter the API at all). Also we don't have plans for another (non-bugfix) 5.x reease at this time. I'd prefer to have this in 6 only for now. Should we then do another 5.x release, it'd be a candidate for backporting of course.